shared/src/commonMain/kotlin/org/hyperskill/app/step/domain/model/Step.kt

```diff
@@ -17,12 +17,18 @@ data class Step(
     val block: Block,
     @SerialName("topic_theory")
     val topicTheory: Long? = null,
+    @SerialName("topic")
+    val topic: Long,
```

Comment:
This is an optional value. In the production, we have more than 1000 of steps with a `null` topic id. -> `val topic: Long? = null`

Reply:
@ivan-magda do we have any API documentation to check potentially nullable types?

Reply:
No, the API documentation doesn't provide that information. We are only able to use backed source code (see a model and its serializer) and Jupyter Notebook (we can execute fetch requests on the database replica).